plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -442,6 +443,7 @@ func ClusterRoles() []rbacv1.ClusterRole {
 				// Needed to check API access.  These creates are non-mutating
 				rbacv1helpers.NewRule("create").Groups(authenticationGroup).Resources("tokenreviews").RuleOrDie(),
 				rbacv1helpers.NewRule("create").Groups(authorizationGroup).Resources("subjectaccessreviews").RuleOrDie(),
+				rbacv1helpers.NewRule(Read...).Groups(schedulingGroup).Resources("priorityclasses").RuleOrDie(),
```

Comment:
since we are denormalizing the preemption policy from the priorityclass into the pod, the scheduler does not need to look at priorityclasses, and this can be dropped

Reply:
Done